src/test/scala/org/graphframes/lib/PageRankSuite.scala

```diff
@@ -19,6 +19,8 @@ package org.graphframes.lib
 
 import org.apache.spark.sql.functions.col
 import org.apache.spark.sql.types.DataTypes
+import org.apache.spark.sql.Row
+import org.apache.spark.ml.linalg.{SQLDataTypes, SparseVector}
```

Comment:
`ParallelPersonalizedPageRank` requires the `SparseVector` type to store the list of PageRanks for each individual source